src/components/settings/SettingsAccessTab/SettingsAccessLinkManagement.jsx

```diff
@@ -130,17 +102,22 @@ const SettingsAccessLinkManagement = ({
         loading={isLoadingLinkManagementEnabledChange}
         disabled={isLoadingLinkManagementEnabledChange}
       >
-        <p>Generate a link to share with your learners.</p>
+        <p>Generate a link to share with your learners (up to a maximum of 100 links).</p>
+        {links.length >= 100 && (
+        <Alert icon={Info} variant="danger">
+          You generated the maximum of 100 links. No additional links may be generated.
```

Comment:
[nit/curious] is it worth creating a constant like `const MAX_UNIVERSAL_LINKS = 100` or similar to use in place of hardcoded `100` throughout? Just thinking ahead to if/when we change the limit in the future.

Reply:
Good idea!